plugins/stages/list.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const jobIdSchema = schema.models.stage.base.extract('jobId');
```

Comment:
We don't have `jobId` in the schema
https://github.com/screwdriver-cd/data-schema/blob/master/models/stage.js#L18
```suggestion
```